snapcraft/tests/test_commands_list_keys.py

```diff
@@ -0,0 +1,104 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2016 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import logging
+from textwrap import dedent
+from unittest import mock
+
+import fixtures
+
+from snapcraft.main import main
+from snapcraft import (
+    storeapi,
+    tests,
+)
+from snapcraft.tests.test_commands_register_key import (
+    get_sample_key,
+    mock_snap_output,
+)
+
+
+class ListKeysTestCase(tests.TestCase):
+
+    scenarios = [
```

Comment:
Is this used somewhere under the hood?  I don't see it referenced anywhere else.